Forms.php

```diff
@@ -606,6 +606,10 @@ protected static function _input_gen( $args ) {
 	 * @return string
 	 */
 	protected static function add_label( $input, $desc, $desc_pos ) {
+		if ( empty( $desc ) ) {
```

Comment:
This won't work since null `desc` will be parsed to empty string here https://github.com/scribu/wp-scb-framework/blob/master/Forms.php#L448. And, probably, it's not necessary to wrap empty string to `label` tag.